src/Compilers/CSharp/Test/Semantic/Semantics/StructsTests.cs

```diff
@@ -614,5 +614,24 @@ public struct X1
     Diagnostic(ErrorCode.ERR_StructsCantContainDefaultConstructor, "X").WithLocation(4, 13)
                 );
         }
+
+        [Fact]
+        public void StructInitializerAutoImplementedProps()
```

Comment:
test name doesn't seem to match the scenario being tested, since this is not an auto-property. Perhaps a name like `StructNonAutoPropertyInitializer`. (don't know if we have a different term for 'non-auto' properties? manual properties?)

Reply:
Name corrected, thanks!